rollingpin/frontends.py

```diff
@@ -63,6 +63,26 @@ def format(self, record):
         return colorize(formatted, color)
 
 
+def generate_component_report(host_results):
+    """Aggregate a list of results from the `components` deploy command."""
+    report = collections.defaultdict(collections.Counter)
+    for host, results in host_results.iteritems():
+        for result in results.get('results', []):
+            if result.command[0] != 'components':
+                continue
+            # Example result.result['components']:
+            #
+            #     {
+            #         'foo': {
+            #             'abcdef': 2,
+            #         },
+            #     }
+            for component, sha_counts in result.result['components'].iteritems():
```

Comment:
let's just bump it to 90 or 100 characters

https://github.com/reddit/styleguide/blob/master/python/pep-0008.rst#maximum-line-length